src/components/List.tsx

```diff
@@ -0,0 +1,9 @@
+type Props = {
+  list: string[];
+};
+
+export const List: React.FC<Props> = ({ list }) => {
+  const listItems = list.map(item => <li key={item}>{item}</li>);
+
+  return <ul>{listItems}</ul>;
```

Comment:
no need for variable, just add inside ul

Reply:
done